static/src/javascripts/lib/fetch-json.ts

```diff
@@ -0,0 +1,39 @@
+import config from 'lib/config';
+
+const fetchJson = (
+	input: RequestInfo,
+	init: RequestInit = {},
+): Promise<unknown> => {
+	// #? we should use Object.assign for this assignment, but this currently breaks
+	// Karma tests that depend on fetch-json.js and have not been stubbed
+	const options = init;
+	let path = typeof input === 'string' ? input : input.url;
+	if (!RegExp('^(https?:)?//').exec(path)) {
+		path = (config.get('page.ajaxUrl', '') as string) + path;
+		options.mode = 'cors';
+		options.credentials = 'include';
+	}
+
+	return fetch(path, options).then((resp) => {
```

Comment:
All calls to `fetch-json` have been checked for compliance to `(input: string, init?: object)`.

```suggestion
	let path = ''
	if (!RegExp('^(https?:)?//').exec(path)) {
		path = String(config.get('page.ajaxUrl', ''));
		init.mode = 'cors';
		init.credentials = 'include';
	}

	return fetch(path + input, init).then((resp) => {
```